rules/aws/stitch-rules-change-description.yaml

```diff
@@ -0,0 +1,22 @@
+kind: Rules
+metadata:
+  namespace: aws.cloudformation
+spec:
+  - name: "ChangeDescriptionAws"
+    condition:
+      deployedType: aws.cloudformation.DeployedTemplate
+    processor:
+      - type: groovy
+        description: "Change description with application name"
```

Comment:
we can change the description to "Change description to constant name", since we are not using the application name